changelog.d/20240306_133305_maria_fix_scope_defining_when_checking_project_permissions.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Scope definition when checking project permissions while requesting status of project import
```

Comment:
This doesn't tell much to someone not familiar with the internals of our IAM system. Can you explain the effect of this change from the perspective of a user?